system-parachains/bridge-hubs/bridge-hub-polkadot/tests/snowbridge.rs

```diff
@@ -196,6 +196,7 @@ fn max_message_queue_service_weight_is_more_than_beacon_extrinsic_weights() {
 	max_message_queue_weight.all_gt(submit_checkpoint);
 }
 
+// FAIL-CI @bkontur can you help me to check why it's exceeding the weight limits?
```

Comment:
@claravanstaden could you please also check this: https://github.com/polkadot-fellows/runtimes/actions/runs/11510473058/job/32042327955#step:9:31? Not sure from the log which extrinsic does not work

Reply:
@bkontur I pulled the latest, looks like it passes now.